### PR TITLE
Add Zone Config Extensions docs

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes DOC-4758

Summary of changes:

- Add new page 'Zone Config Extensions', which describes the feature at a high level

- Add new SQL syntax page `ALTER DATABASE ... ALTER LOCALITY`, which shows how to do some stuff

- Add mention of 'Zone Config Extensions' to the 'Multi-Region Capabilities Overview' page

- Add links to 'Zone Config Extensions' from various multi-region-related pages around the docs

- Make opportunistic edit to zone config docs explaining what `DISCARD` does, based on description from CRDB in `pkg/sql/set_zone_config.go`.

### CODE REVIEW

**Comment by rmloveland** [comment]
Hi Jane and Nathan!

Any feedback on these changes?  I hope they're pretty close to accurate due to having Nathan's Google doc and the logic tests to work from.

If you're pressed for time the substantive additions in this PR are:

- a few paras on a new 'Zone Config Extensions' page which has some overview info
- a new 'ALTER DB .. ALTER LOCALITY' SQL statement page which has like 1-2 examples
- a very small clarification of how DROP vs RESET zone configs works that appears to have been previously undocumented (based on something I found in the code while working on this)

Everything else is just infrastructure stuff like adding links etc.

**Review by nvanbenschoten — COMMENTED** [review]
> Reviewed 16 of 17 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15304)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @rmloveland and @ZhouXing19)

___
*[`v22.2/alter-locality.md` line 27 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NF9YadH0IKD_62SUE-z:-NF9YadH0IKD_62SUE0-:b-68jgx2) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/alter-locality.md#L27)):*
> ```Markdown
> |-------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
> | `database`        | The [multi-region database](multiregion-overview.html)  containing the [regions](multiregion-overview.html#database-regions) whose [zone configs](configure-replication-zones.html) you want to modify.                                                |
> | `locality_scope`  | One of: `GLOBAL` ([global tables](multiregion-overview.html#global-tables)), `REGIONAL` ([regional tables](multiregion-overview.html#regional-tables)), or `REGIONAL IN` ([regional by row tables](multiregion-overview.html#regional-by-row-tables)). |
> ```

This isn't quite correct. `REGIONAL` refers to all `REGIONAL BY TABLE` and `REGIONAL BY ROW` tables. In other words, all "regional" data. `REGIONAL IN <region>` refers to all `REGIONAL BY TABLE IN <region>` tables and all `<region>` partitions of `REGIONAL BY ROW` tables. In other words, all "regional" data homed in <region>.
___
*[`v22.2/alter-locality.md` line 154 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NF9ZGbz04vlAZsxZuOu:-NF9ZGbz04vlAZsxZuOv:bagmkae) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/alter-locality.md#L154)):*
> ```Markdown
> 
> ~~~
> ALTER DATABASE ALTER LOCALITY CONFIGURE ZONE
> ```

These are outputs from the previous `ALTER DATABASE` statement, right? Are they necessary? I found them to be noisy and mildly confusing when reading the page.
___
*[`v22.2/alter-locality.md` line 168 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NEaoc4C3DOrgCK630we:-NF9ZgVX0ipHy1Mh4cRb:bgb29y9) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/alter-locality.md#L168)):*
<details><summary><i>Previously, ZhouXing19 (Jane Xing) wrote…</i></summary><blockquote>

I don't know if we want this and if this is even consistent with _"configure `us-west1` to be the failover region"_. In the above example we didn't show tables with primary region in `europe-west1`, so is it necessary for us to have this part? Maybe we can create some tables with region specified, and also do `SHOW ZONE CONFIGURATION FOR TABLE` before and after running `ALTER DATABASE ALTER LOCALITY`?

Also, if we want `us-west1` to be the failover region, why add `us-east1` to the voter constraints and lease preferences here?

cc @nvanbenschoten to confirm if I have understood it correctly. Thanks!

</blockquote></details>

I agree that this is configuring `us-east1` to be the failover region. That makes sense for `europe-west1` (the east coast of the US is closer), but wasn't explained above.
___
*[`v22.2/zone-config-extensions.md` line 20 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NF9XhedDEfHHiHLUJ5B:-NF9XhedDEfHHiHLUJ5C:b-m7lc6) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/zone-config-extensions.md#L20)):*
> ```Markdown
> - Data domiciling
> 
> We strongly recommend that most users use the [multi-region SQL abstractions](multiregion-overview.html) as much as possible. However, if you need additional customizations, using Zone Config Extensions is better than writing your own low-level [zone configs](configure-replication-zones.html) for the following reasons:
> ```

Users are still using the multi-region SQL abstractions when using these extensions, so is this the right way to phrase the sentence? Maybe "use the default multi-region SQL abstractions"?
___
*[`v22.2/zone-config-extensions.md` line 22 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NF9Y0ekBJvalisIpUOz:-NF9Y0ekBJvalisIpUP-:b2xqsrt) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/zone-config-extensions.md#L22)):*
> ```Markdown
> We strongly recommend that most users use the [multi-region SQL abstractions](multiregion-overview.html) as much as possible. However, if you need additional customizations, using Zone Config Extensions is better than writing your own low-level [zone configs](configure-replication-zones.html) for the following reasons:
> 
> - Zone Config Extensions are persistent and kept up-to-date. Unlike raw zone configs, which must be fully overwritten on each update in order to take effect, Zone Config Extensions stay in place alongside the high-level [multi-region SQL configuration](multiregion-overview.html). This is true even if you [add new tables](create-table.html) or [regions](add-region.html), or if a table's [locality is changed](set-locality.html).
> ```

"raw zone config modification"

"on each multi-region configuration change"
___
*[`v22.2/zone-config-extensions.md` line 25 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NF9YGdp58lgHKS-wn7b:-NF9YGdp58lgHKS-wn7c:b-icn5vb) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/zone-config-extensions.md#L25)):*
> ```Markdown
> - They are validated against the configs already in place in a multi-region cluster. Unlike hand-written zone configs, they are designed to work properly with the configuration generated by the higher-level multi-region SQL statements.
> - They are composable. You can use Zone Config Extensions to build up approximations of many higher-level features. For an example, see [Failover regions](alter-locality.html#failover-regions).
> - They are region scoped. When a Zone Config Extension is attached to a [database region](multiregion-overview.html#database-regions), any [`REGIONAL BY TABLE`](multiregion-overview.html#regional-tables) or [`REGIONAL BY ROW`](multiregion-overview.html#regional-by-row-tables) tables associated with that region will have their zone configurations updated according to the settings passed via the extension. As mentioned above, this updated config will also be persisted in the face of other multi-region configuration changes.
> ```

They are also locality scoped. So for instance, you can specify a zone config extension that only applies to GLOBAL tables.

**Review by rmloveland — COMMENTED** [review]
Jane and Nathan - made updates based on your feedback - PTAL!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15304#-:-NFKl3yj0hXk3TRjq3U6:b605sl)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @nvanbenschoten and @ZhouXing19)

___
*[`v22.2/alter-database.md` line 27 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NEajipp5a771CHLBBgu:-NFFXFarD_6Nj5v2tAcI:bhioh0d) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/alter-database.md#L27)):*
<details><summary><i>Previously, ZhouXing19 (Jane Xing) wrote…</i></summary><blockquote>

I think this syntax was added since v22.2 -- Nathan added the descriptor for the zone config extension in v22.1, but it's since v22.2 that we allow users to configure it.
</blockquote></details>

ah, copy-paste error - thanks! updated to say v22.2
___
*[`v22.2/alter-locality.md` line 27 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NF9YadH0IKD_62SUE-z:-NFJpldz0Soy2pyu-_Oj:b-wjjpjv) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/alter-locality.md#L27)):*
<details><summary><i>Previously, nvanbenschoten (Nathan VanBenschoten) wrote…</i></summary><blockquote>

This isn't quite correct. `REGIONAL` refers to all `REGIONAL BY TABLE` and `REGIONAL BY ROW` tables. In other words, all "regional" data. `REGIONAL IN <region>` refers to all `REGIONAL BY TABLE IN <region>` tables and all `<region>` partitions of `REGIONAL BY ROW` tables. In other words, all "regional" data homed in <region>.
</blockquote></details>

updated with ~this info, PTAL
___
*[`v22.2/alter-locality.md` line 34 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NEapO8X2dflBC0Mb8Sj:-NFFjnDD68n3vCMNisgs:b-4aeo3o) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/alter-locality.md#L34)):*
<details><summary><i>Previously, ZhouXing19 (Jane Xing) wrote…</i></summary><blockquote>

I wonder if we can provide some more general examples before diving into use cases with secondary regions etc.
For example (I'm copy-pasting from the existing logic test so revision will be needed if we want to use them):

```
query TT
SHOW ZONE CONFIGURATION FOR TABLE tbl
----
DATABASE "mr-zone-configs"  ALTER DATABASE "mr-zone-configs" CONFIGURE ZONE USING
                            range_min_bytes = 134217728,
                            range_max_bytes = 536870912,
                            gc.ttlseconds = 90000,
                            num_replicas = 9,
                            num_voters = 5,
                            constraints = '{+region=ap-southeast-2: 1, +region=ca-central-1: 1, +region=us-east-1: 1}',
                            voter_constraints = '{+region=ca-central-1: 2}',
                            lease_preferences = '[[+region=ca-central-1]]'

statement ok
ALTER DATABASE "mr-zone-configs" ALTER LOCALITY REGIONAL IN "ca-central-1" CONFIGURE ZONE USING
  lease_preferences = '[[+region=ca-central-1], [+region=ap-southeast-2]]'

query TT
SHOW ZONE CONFIGURATION FOR TABLE tbl
----
DATABASE "mr-zone-configs"  ALTER DATABASE "mr-zone-configs" CONFIGURE ZONE USING
                            range_min_bytes = 134217728,
                            range_max_bytes = 536870912,
                            gc.ttlseconds = 90000,
                            num_replicas = 9,
                            num_voters = 5,
                            constraints = '{+region=ap-southeast-2: 1, +region=ca-central-1: 1, +region=us-east-1: 1}',
                            voter_constraints = '{+region=ca-central-1: 2}',
                            lease_preferences = '[[+region=ca-central-1], [+region=ap-southeast-2]]'
```
Essentially, i think we may want to deliver a clear and relatively simple idea that zone config extension is used to override the specified fields (in this example, `lease_preferences` ) of certain schema objects' zone configs.

</blockquote></details>

Good idea!

I have added a basic 'override specific fields of a zone config' example based on the idea from the logic test you provided but using the movr setup already in the doc

It only updates `lease_preferences` as shown in the logic test to add 1 more region

Please let me know what you think!
___
*[`v22.2/alter-locality.md` line 154 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NF9ZGbz04vlAZsxZuOu:-NFFkAMZ6vafGudxzAHP:b-dab9en) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/alter-locality.md#L154)):*
<details><summary><i>Previously, nvanbenschoten (Nathan VanBenschoten) wrote…</i></summary><blockquote>

These are outputs from the previous `ALTER DATABASE` statement, right? Are they necessary? I found them to be noisy and mildly confusing when reading the page.
</blockquote></details>

yes they are the outputs

I agree with your opinion tho, and don't really think they are necessary

originally added them because it's kind of a convention throughout the docs to add them, but tbh I don't think that convention adds value

went ahead and filed https://cockroachlabs.atlassian.net/browse/DOC-6036 so the docs style team can discuss and hopefully we can just ... stop doing this

that said no need to litigate with style team in this PR

removed them from everywhere on this page I could find
___
*[`v22.2/alter-locality.md` line 168 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NEaoc4C3DOrgCK630we:-NFK19As1EKWVF3HTJrL:b-p09fr2) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/alter-locality.md#L168)):*
<details><summary><i>Previously, nvanbenschoten (Nathan VanBenschoten) wrote…</i></summary><blockquote>

I agree that this is configuring `us-east1` to be the failover region. That makes sense for `europe-west1` (the east coast of the US is closer), but wasn't explained above.
</blockquote></details>

"Maybe we can create some tables with region specified, and also do SHOW ZONE CONFIGURATION FOR TABLE before and after running ALTER DATABASE ALTER LOCALITY?"

Great idea, thanks for the feedback! Revised the example to:

1. show the zone configs for the `movr.rides` table
2. alter the `movr.rides` table to make it `REGIONAL BY ROW` and set appropriate home regions based on value of `city` columns (Amsterdam = EU, etc)
2. only do 1 zone config extension change: set the failover region for `REGIONAL IN "us-east1"` to `us-west1`
3. then, show the `movr.rides` zone configs again to show what changed and describe that

And removed the stuff where I was configuring additional failover regions for EU, us-west

hopefully it's a little clearer now - PTAL!
___
*[`v22.2/zone-config-extensions.md` line 20 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NF9XhedDEfHHiHLUJ5B:-NFFqzDF22ZmGCWMm-31:britydw) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/zone-config-extensions.md#L20)):*
<details><summary><i>Previously, nvanbenschoten (Nathan VanBenschoten) wrote…</i></summary><blockquote>

Users are still using the multi-region SQL abstractions when using these extensions, so is this the right way to phrase the sentence? Maybe "use the default multi-region SQL abstractions"?
</blockquote></details>

Ah, good clarification - updated to say "use the default .. as much as possible."
___
*[`v22.2/zone-config-extensions.md` line 22 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NF9Y0ekBJvalisIpUOz:-NFFrAuP7A_Zw_5KJKrr:b-utpun5) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/zone-config-extensions.md#L22)):*
<details><summary><i>Previously, nvanbenschoten (Nathan VanBenschoten) wrote…</i></summary><blockquote>

"raw zone config modification"

"on each multi-region configuration change"
</blockquote></details>

check and check!
___
*[`v22.2/zone-config-extensions.md` line 23 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NEarmXy0L0hfP1evRqB:-NFJqohgAD0ZaoeepKPk:bax5jop) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/zone-config-extensions.md#L23)):*
Thanks for sharing that PR, very helpful!

Based on the PR description, added the following line to this doc:

"For example, they will not change the [lease preferences](configure-replication-zones.html#lease_preferences) or [home regions](multiregion-overview.html#regional-by-row-tables) for Zone Config Extensions scoped to the [`REGIONAL` or `REGIONAL IN` locality scope](alter-locality.html#parameters), nor the [survival goals](multiregion-overview.html#survival-goals) for the database."

>  It would be good if we can include certain examples for this too.

For length reasons, I would rather keep the examples focused on the happy path and let the validation logic in the DB show users an error when they try to do something they should not do
___
*[`v22.2/zone-config-extensions.md` line 25 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NF9YGdp58lgHKS-wn7b:-NFJmfSC0Wso6K5mpet5:bxwoato) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/zone-config-extensions.md#L25)):*
<details><summary><i>Previously, nvanbenschoten (Nathan VanBenschoten) wrote…</i></summary><blockquote>

They are also locality scoped. So for instance, you can specify a zone config extension that only applies to GLOBAL tables.
</blockquote></details>

Thanks, added some words about this in another bullet

**Review by ZhouXing19 — APPROVED** [review]
Nice work! :lgtm: modules some nits. 
@nvanbenschoten could you finalize the review? Thanks!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15304#-:-NGNgFJ8AN_wG1jxyUGO:bqdc6lr)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @nvanbenschoten and @rmloveland)

___
*[`v22.2/alter-locality.md` line 34 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NEapO8X2dflBC0Mb8Sj:-NGMzcGP5XFMOfxQe3jc:b-qbxl94) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/alter-locality.md#L34)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

Good idea!

I have added a basic 'override specific fields of a zone config' example based on the idea from the logic test you provided but using the movr setup already in the doc

It only updates `lease_preferences` as shown in the logic test to add 1 more region

Please let me know what you think!
</blockquote></details>

Looks great, thanks!
___
*[`v22.2/alter-locality.md` line 60 at r2](https://reviewable.io/reviews/cockroachdb/docs/15304#-NGNeYY1Djx7gpP6GBWg:-NGNeYY1Djx7gpP6GBWh:b-267tw4) ([raw file](https://github.com/cockroachdb/docs/blob/44224320d1a2b9ba227c5c7525408f090f7142f9/v22.2/alter-locality.md#L60)):*
> ```Markdown
> ~~~
> 
> This gives us a [MovR](movr.html) database with the following regions according to [`SHOW REGIONS`](show-regions.html):
> ```

This is a tiny nit: this sentence sounds like we already have `MovR` a multi-region db, but the title below says `Make the database multi-region`, and it can be a bit confusing.

I wonder if we can rephrase this to `This gives us a cluster with the following regions, and a (preloaded) database MovR`.
___
*[`v22.2/zone-config-extensions.md` line 23 at r1](https://reviewable.io/reviews/cockroachdb/docs/15304#-NEarmXy0L0hfP1evRqB:-NGN4jgiB_EcRgi_zyA6:b3akwgl) ([raw file](https://github.com/cockroachdb/docs/blob/c4912bac093b6963a219cc3f9f7bee7b3e9fed50/v22.2/zone-config-extensions.md#L23)):*
> they will not change the lease preferences or home regions for Zone Config Extensions scoped to the REGIONAL or REGIONAL IN locality scope,

This sentence can be a bit confusing, extensions with REGIONAL IN can modified lease preferences, just not in contradiction with the home region.

So can we be more explicit and say 
- REGIONAL extensions don't set lease_preferences
- REGIONAL IN extensions don't set lease_preferences, replica constraints or voter contains that disagree with the original home region

**Comment by rmloveland** [comment]
Thanks for the helpful and thorough reviews @ZhouXing19 and @nvanbenschoten !

Any additional updates needed before this goes on to Docs team review?

**Comment by rmloveland** [comment]
Thanks for the reviews Jane and Nathan!

Moving this one along for Docs team review by @ianjevans

File: v22.2/alter-locality.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+
+~~~
+ALTER DATABASE <database> ALTER LOCALITY <locality_scope> CONFIGURE ZONE USING <set_zone_config>;
+~~~
+
+## Parameters
+
+Parameter | Description
+ --- | ---
+`database` | The [multi-region database](multiregion-overview.html)  containing the [regions](multiregion-overview.html#database-regions) whose [zone configs](configure-replication-zones.html) you want to modify.
```
Comment (ianjevans) [inline v22.2/alter-locality.md:26]:
```suggestion
`database` | The [multi-region database](multiregion-overview.html) containing the [regions](multiregion-overview.html#database-regions) whose [zone configs](configure-replication-zones.html) you want to modify.
```

File: v22.2/alter-locality.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
+---
+title: ALTER LOCALITY
+summary: The ALTER LOCALITY statement applies Zone Config Extensions to a multi-region database.
+toc: true
+docs_area: reference.sql
+---
+
+<span class="version-tag">New in v22.2:</span> The `ALTER DATABASE .. ALTER LOCALITY` [statement](sql-statements.html) provides a customization tool for power users to persistently modify the configuration generated by the standard [multi-region SQL abstractions](multiregion-overview.html) on a per-region basis.
```
Comment (ianjevans) [inline v22.2/alter-locality.md:8]:
```suggestion
<span class="version-tag">New in v22.2:</span> The `ALTER DATABASE .. ALTER LOCALITY` [statement](sql-statements.html) provides a customization tool for advanced users to persistently modify the configuration generated by the standard [multi-region SQL abstractions](multiregion-overview.html) on a per-region basis.
```

File: v22.2/multiregion-overview.md [hunk 3]
```diff
@@ -216,6 +216,12 @@ For an example that uses unique indexes but applies to all indexes on `REGIONAL
 
 Regional by row tables can take advantage of [hash-sharded indexes](hash-sharded-indexes.html) provided the `crdb_region` column is not part of the columns in the hash-sharded index.
 
+### Zone config extensions
+
+Zone Config Extensions are a customization tool for power users to persistently modify the configuration generated by the standard [multi-region SQL abstractions](multiregion-overview.html) on a per-region basis.
```
Comment (ianjevans) [inline v22.2/multiregion-overview.md:221]:
```suggestion
Zone Config Extensions are a customization tool for advanced users to persistently modify the configuration generated by the standard [multi-region SQL abstractions](multiregion-overview.html) on a per-region basis.
```

File: v22.2/zone-config-extensions.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+title: Multi-Region Zone Config Extensions
+summary: Multi-Region Zone Config Extensions provide flexibility to multi-region SQL abstractions.
+toc: true
+keywords: multiregion, region, regions, zone config, zone configs, configure replication zones, replication zone, replications zones, zcfg, zcfgs
+docs_area: reference
+---
+
+<span class="version-tag">New in v22.2:</span> Multi-Region _Zone Config Extensions_ are a customization tool for power users to persistently modify the configuration generated by the standard [multi-region SQL abstractions](multiregion-overview.html) on a per-region basis.
```
Comment (ianjevans) [inline v22.2/zone-config-extensions.md:9]:
```suggestion
<span class="version-tag">New in v22.2:</span> Multi-Region _Zone Config Extensions_ are a customization tool for advanced users to persistently modify the configuration generated by the standard [multi-region SQL abstractions](multiregion-overview.html) on a per-region basis.
```

**Review by ianjevans — APPROVED** [review]
Looks good, just a couple nit suggestions.

**Review by nickvigilante — APPROVED** [review]
Approved netlify/vale/vocab.txt only
